lang/go/package_test.go

```diff
@@ -15,14 +15,15 @@ func TestPackageName(t *testing.T) {
 		dir      string
 		expected pgs.Name
 	}{
-		{"keyword", "_package"},        // go keywords are prefixed with _
-		{"none", "NO_pack__age_name_"}, // if there is no package or go_package option, use the input filepath
-		{"package", "my_package"},      // use the go_package option
-		{"unnamed", "names_unnamed"},   // use the proto package if no go_package
-		{"import", "bar"},              // uses the basename if go_package contains a /
-		{"override", "baz"},            // if go_package contains ;, use everything to the right
-		{"import_path", "_package"},    // import_path param used if no go_package option
-		{"mapped", "unaffected"},       // M mapped params are ignored for build targets
+		{"keyword", "_package"},              // go keywords are prefixed with _
+		{"none", "NO_pack__age_name_"},       // if there is no package or go_package option, use the input filepath
+		{"package", "my_package"},            // use the go_package option
+		{"unnamed", "names_unnamed"},         // use the proto package if no go_package
+		{"import", "bar"},                    // uses the basename if go_package contains a /
+		{"override", "baz"},                  // if go_package contains ;, use everything to the right
+		{"import_path", "_package"},          // import_path param used if no go_package option
+		{"mapped", "unaffected"},             // M mapped params are ignored for build targets
+		{"import_path_mapped", "go_package"}, // mixed import_path and M parameters should lose to go_package
```

Comment:
@rodaine what does this change do, is it just testing something?

Reply:
It tests that the officially generated package name of import_path_mapped.proto matches the one expected (in this case `go_package`).

Reply:
Most of the "changes" you're seeing here is whitespace from the comment realignment (via gofmt)